charts/et-hearings-api/values.yaml

```diff
@@ -4,10 +4,10 @@ java:
   ingressHost: et-hearings-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: et
   keyVaults:
-    et-hearings-api:
-      secrets:
-        - name: et-hearings-api-s2s-secret
-          alias: ET_HEARINGS_S2S_SECRET
+#    et-hearings-api:
```

Comment:
Is this intentional?

Reply:
shouldn't need to comment out these, the latest updates to ecm-ccd-docker should allow master to work without doing that.

Reply:
Reverted